backend/scripts/add_price_and_bedroom_attributes.ts

```diff
@@ -0,0 +1,19 @@
+import { Review } from '@common/types/db-types';
+import { db } from '../src/firebase-config';
+
+// This script will add the price and bedrooms attributes to all the reviews that do not have them
```

Comment:
For the future, you should use a docstring/multi line documentation to document entire components or scripts. You should only use `//` for documenting specific lines of code. For example for this script you can write:
```
/** 
* This script will add the price and bedrooms attributes to all the reviews that do not have them
*/
```